x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/create_analytics_form/job_type.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment, FC } from 'react';
+import { i18n } from '@kbn/i18n';
+
+import { EuiFormRow, EuiSelect } from '@elastic/eui';
+
+import { AnalyticsJobType, JOB_TYPES } from '../../hooks/use_create_analytics_form/state';
+
+interface Props {
+  type: AnalyticsJobType;
+  setFormState: any; // TODO update type
+}
+
+export const JobType: FC<Props> = ({ type, setFormState }) => {
```

Comment:
Nit: Suggest to give this a more specific name, e.g. `FormRowJobType` / `form_row_job_type.tsx`.